flocker/node/agents/blockdevice.py

```diff
@@ -543,6 +569,79 @@ def _valid_size(size):
 
 @implementer(IStateChange)
 @provider(IDatasetStateChangeFactory)
+class CreateMountSymlink(PClass):
```

Comment:
This is starting to feel very Java-y :crying_cat_face: Not worth fixing here though.